docs/topics/mta-rn-known-issues-6-2-0.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-6-2-0_{context}"]
+= Known issues
+
+MTA version 6.2.0 has the following issues.
+
+.MTA fails to run an analysis of private GitHub repos
+
+Analysis of private GitHub repositories fails. link:https://issues.redhat.com/browse/MTA-810[MTA-810]
+
+.Analysis fails when fetching rules from a repository with a folder that contains spaces in its name
+
+When fetching custom rules from a repository during an analysis, if the `Root path` field contains spaces,  the `mta-cli` command is not properly composed and the analysis fails. link:https://issues.redhat.com/browse/MTA-458[MTA-458]
+
+.Update notifications are disabled for Application, Job functions, and Business services
+
+Update notifications are disabled for **Application**, **Job function** and **Business services**, as a result, no notifications are displayed. link:https://issues.redhat.com/browse/MTA-1042[MTA-1042]
+
+.Repository type field is not required
+
+The **Repository** field is not required when saving the configuring rules files from a repository in analysis. link:https://issues.redhat.com/browse/MTA-1047[MTA-1047]
+
+.Connection status is slow when moving from Not connected to Connected
```

Comment:
@anarnold97 I am not sure this is the case, @ibolton336 could you please confirm?